src/pages/api/delete-identity.ts

```diff
@@ -17,10 +18,7 @@ import getTracer from "@src/tracer";
 
 const tracer = getTracer();
 
-const handler = (
-  request: NextApiRequest,
-  response: NextApiResponse,
-): Promise<void> => {
+const handler: Handler = (request, response): Promise<void> => {
   return tracer.span("delete-identity handler", async (span) => {
     const { userId, type, value } = request.body;
 
```

Comment:
> span.setDisclosedAttribute("Identity type", type);

Maybe we don't need this one anymore ?

Reply:
I'm really 50-50 on this one.

Reply:
Why don't we need it?

Reply:
I thought it was some kind of test during first implementation. What are the benefits of keeping it ? 🤔

Reply:
To give information when you're looking at a trace of what is happening without having to guess. In this case, you're looking at the `delete-identity` handler, you can see on the right (on lightstep) that the identity you're trying to delete is of type `EMAIL` or `PHONE` or something.

Isn't it a good info?

Reply:
I suppose it could be a useful info indeed, I clearly lack experience with traces and using tools as Lightstep. 😅

Reply:
When in doubt, leave more info :)